pkgs/applications/networking/seadrive-gui/default.nix

```diff
@@ -0,0 +1,60 @@
+{ cmake
+, fetchFromGitHub
+, makeWrapper
+, pkgconfig
+, stdenv
+# Package dependencies
+, libsearpc
+, libselinux
+, libsepol
+, libuuid
+, pcre
+, qttools
+, qtwebengine
+, seadrive-daemon
+}:
+
+stdenv.mkDerivation rec {
+  pname = "seadrive-gui";
+  version = "1.0.7";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1dm7f2d631rjlhhvfnsvm8p749gqrcvjpqxi5gbhfq4hxbbn5xmd";
+  };
+
+  patches = [ ./fix_webkit_webengine.patch ];
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+    pkgconfig
+  ];
+  buildInputs = [
+    libsearpc
+    libselinux
+    libsepol
+    libuuid # Satisfies the 'mount' package requirement. Contains 'mount.pc'
+    pcre
+    qttools
+    qtwebengine
+    seadrive-daemon
+  ];
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
+
+  postInstall = ''
+    makeWrapper ${seadrive-daemon}/bin/seadrive $out/bin/seadrive
```

Comment:
Do you need a wrapper here instead of just a copy or symlink

Reply:
`makeWrapper` was recommended over at the #nixos irc. I'm not sure why it was recommended over a symlink.

Reply:
`seadrive-gui` calls `seadrive-daemon/bin/seadrive` at runtime. I will add a comment.

Reply:
In that case I understand why you were recommended to use `makeWrapper`. Try wrapping the `seadrive-gui` binary and passing the `seadrive-daemon` binary as an argument via `--prefix PATH` like so:
```
wrapProgram $out/bin/seadrive --prefix PATH : ${lib.makeBinPath [ seadrive-daemon ]}
```

Reply:
Thank you. Looks like `wrapProgram` works just as well as `makeWrapper`, and doesn't pollute the `seadrive-gui` bin.